wire-ios/Wire-iOS/Sources/Authentication/Interface/Views/ValidatedTextField.swift

```diff
@@ -335,9 +339,12 @@ final class ValidatedTextField: AccessoryTextField, TextContainer, Themeable {
         validateInput()
     }
 
-    func validateInput() {
-        let error = textFieldValidator.validate(text: text, kind: kind)
+    @discardableResult
+    func validateInput(useGuestLinkRuleset: Bool = false) -> Bool {
+        let error = textFieldValidator.validate(text: text, kind: kind, useGuestLinkRuleset: useGuestLinkRuleset)
```

Comment:
you can check if the error is nil, you can return the result and not continue the calculation.

Reply:
Done!